### ISSUE TITLE
prisma deploy fails in version 1.22.0

### ISSUE THREAD

**Issue by ajmueller** [prisma/prisma#3645]
**Describe the bug**
Three days ago I built a new Prisma Server following the [Set up on a new database tutorial](https://www.prisma.io/docs/get-started/01-setting-up-prisma-new-database-JAVASCRIPT-a002/).  That was on version 1.21.0.  Today, going back to update my schema, `prisma deploy` fails with the CLI@1.22.0.  I've updated my Docker container to version 1.22.0 as well to no avail.

**To Reproduce**
1. Go to [Set up on a new database tutorial](https://www.prisma.io/docs/1.21/get-started/01-setting-up-prisma-new-database-JAVASCRIPT-a002/).
2. Follow the tutorial and start your Prisma Server.
3. Run `prisma deploy` with version 1.21.0 of the CLI.  It should run successfully.
4. Upgrade the CLI to 1.22.0.
5. Run `prisma deploy` and it will fail with the below console output.

**Expected behavior**
The `deploy` command to run properly.

**Console Output**
```shell
$ prisma deploy
 ▸    Error parsing plugin
 ▸    /snapshot/cli/node_modules/prisma-cli-core

 ▸    'MODULE_NOT_FOUND': Cannot find module 'copy-paste'
 ▸    1) If you want to compile the package/file into executable,
 ▸    please pay attention to compilation warnings and specify a
 ▸    literal in 'require' call. 2) If you don't want to compile the
 ▸    package/file into executable and want to 'require' it from
 ▸    filesystem (likely plugin), specify an absolute path in
 ▸    'require' call using process.cwd() or process.execPath.

 ▸    Error parsing plugin
 ▸    /snapshot/cli/node_modules/prisma-cli-core

 ▸    'MODULE_NOT_FOUND': Cannot find module 'copy-paste'
 ▸    1) If you want to compile the package/file into executable,
 ▸    please pay attention to compilation warnings and specify a
 ▸    literal in 'require' call. 2) If you don't want to compile the
 ▸    package/file into executable and want to 'require' it from
 ▸    filesystem (likely plugin), specify an absolute path in
 ▸    'require' call using process.cwd() or process.execPath.

 ▸    Error parsing plugin
 ▸    /snapshot/cli/node_modules/prisma-cli-core

 ▸    'MODULE_NOT_FOUND': Cannot find module 'copy-paste'
 ▸    1) If you want to compile the package/file into executable,
 ▸    please pay attention to compilation warnings and specify a
 ▸    literal in 'require' call. 2) If you don't want to compile the
 ▸    package/file into executable and want to 'require' it from
 ▸    filesystem (likely plugin), specify an absolute path in
 ▸    'require' call using process.cwd() or process.execPath.

 ▸    deploy is not a prisma command.
 ▸    Perhaps you meant help
 ▸    Run prisma help for a list of available
 ▸    commands.

Get in touch if you need help: https://www.prisma.io/forum/
To get more detailed output, run $ export DEBUG="*"
```

**Versions (please complete the following information):**
 - OS: macOS Mojave
 - `prisma` CLI: `prisma/1.22.0 (darwin-x64)`
 - Prisma Server: `1.22.0`
 - `prisma-binding`: `^2.2.8`

**Comment by timsuchanek** [comment]
Thanks for reporting @ajmueller ! The problem is, that an old version of one of the cli packages (`prisma-cli-core`) is being loaded. We're using an internal cache for packages, which in this case apparently doesn't get invalidated.

How did you install the `prisma` CLI? With `npm install -g prisma`?

2 fixes come to my mind:
1. You can try to set the env var `GRAPHCOOL_CLI_CLEAR_CACHE` to `true` and run the prisma cli again.
2. Reinstall the prisma cli.

**Comment by ajmueller** [comment]
@timsuchanek sorry for the delayed response; I ran into this on a side project that I haven't had a chance to work on again until today.

I installed Prisma via Homebrew.  As such, I used `brew upgrade prisma` to upgrade from 1.21.0 to 1.22.0.  For some strange reason this morning, 1.22.0 is working just fine and I don't know why.  I changed both the Prisma CLI and Prisma Server to 1.22.0 just as I did the other day and the `deploy` command is working perfectly 😕

I see that 1.22.2 is out, but it's not being recognized when I now run `brew update` followed by `brew upgrade prisma`.  I'm not concerned, though, since 1.22.0 is now working for me.

Sorry for the false alarm and thanks for doing such awesome work!  I'm just starting to get into GraphQL and you guys do such an amazing job to make it easy to get started.
